src/main/java/uk/gov/justice/digital/client/glue/GlueClient.java

```diff
@@ -132,6 +131,7 @@ private List<Column> getColumnsAndModifyTypes(StructType schema) {
                     break;
             }
 
+            if (keyColumnNames.contains(column.getName())) column.setComment("primary_key");
```

Comment:
This sets the comment to `primary_key` when the column is included in the source reference primary keys.